src/Xamarin.Android.Build.Tasks/Tasks/GetAotArguments.cs

```diff
@@ -149,7 +149,7 @@ int GetNdkApiLevel (NdkTools ndk, AndroidTargetArch arch)
 				// level already set
 			} else {
 				// Probably not ideal!
-				level       = MonoAndroidHelper.SupportedVersions.MaxStableVersion.ApiLevel;
+				level       = MonoAndroidHelper.SupportedVersions.MaxStableVersion?.ApiLevel ?? 0;
```

Comment:
Is there a different default we can use? using 0 here would cause a very odd error for the user I would guess.

Reply:
I guess `21` would be better for now.